lib/logout.js

```diff
@@ -175,12 +175,12 @@ module.exports.logout = function (options) {
             return next(new Error('SAML Request with no issuer. Issuer is a mandatory element.'));
           }
 
-          options.sessionParticipants.get(requestData.issuer, requestData.sessionIndex, requestData.nameId, function (err, session) {
+          options.sessionParticipants.get(requestData.issuer, requestData.sessionIndices, requestData.nameId, function (err, session) {
```

Comment:
I'm not suggesting to do this but could we also call `sessionParticipants.get` repeatedly to achieve the same effect while retaining backward compatibility?

Reply:
That should work. I initially explored this option, though we'd have to do that everywhere we call `#get`.

I'm guessing when the function definitions were written, multiple session indices in a LogoutRequest may have not been considered (judging by the JSDoc on `#get`) which is why I tried to take the opportunity to simplify it.

Also, if a developer's custom implementation of `#get` requires a call to the DB, it may result in additional overhead which also nudged me in the direction of a single call with an array of session indices.